Topics/User_Experience.md

```diff
@@ -21,9 +21,18 @@ ___Established Norms___
 ___Color Theory___
 - [Colors Influence Choice](https://usabilitygeek.com/colour-user-experience-psychology/#:~:text=Colour%20plays%20a%20crucial%20role,and%20identified%20with%20your%20industry.): Users are shown certain colors to enforce actions. For example, red is commonly linked to aggressive or bad emotions and as such can be used as a cancel to dissuade people from refunding items.
 
+___Responsive Design___
+- [Responsive Design](https://devrix.com/tutorial/important-responsive-design/): Demonstrates important and benefits of responsive design, flexiblity they provide, easiblity for the consumers upon making the said software. 
```

Comment:
Nice resource on responsive design and a summary of what it is about. Could be helpful to provide a brief sentence explaining what responsive design is, and why it might be helpful to a CSC301 student.

Reply:
Thank you. Good idea! I have added the explanation and its benefits of responsive design for a general piece of software.

Reply:
Looks good!